doc/build.md

```diff
@@ -16,12 +12,17 @@ to update all `BUILD` files:
 
     gazelle -go_prefix "istio.io/manager" -mode fix -repo_root .
 
-[Gazelle tool](https://github.com/bazelbuild/rules_go/tree/master/go/tools/gazelle) to automatically generate `BUILD` files ships with Bazel and is located in Bazel workspace cache:
+Install [Gazelle tool](https://github.com/bazelbuild/rules_go/tree/master/go/tools/gazelle) that automatically generates Bazel `BUILD` files as follows:
+
+    go install github.com/bazelbuild/rules_go/go/tools/gazelle/gazelle
+
+_Note_: Gazelle incorrectly rewrites Istio API dependency. You need to manually replace:
```

Comment:
This note doesn't say _where_ this needs to be done. Perhaps we put line 13 and this fix-up into a script to abstract these details away, e.g. `bin/gazelle.sh` ?